index.json

```diff
@@ -749,7 +800,8 @@
       "currentSpecification": "background-fetch"
     },
     "nightly": {
-      "url": "https://wicg.github.io/background-fetch/"
+      "url": "https://wicg.github.io/background-fetch/",
+      "repository": "https://github.com/wicg/background-fetch"
```

Comment:
not sure how to deal with it, nor how strongly we should care, but for WICG, WebAssembly, WebBluetoothCG, WebAudio, the mapping algorithm lowercase the path of the org name - github redirects the URL, but this will likely make reusing the data more difficult

Reply:
Not sure how to deal with it either, I note specs are also inconsistent in the way they link to the repository. For instance, Background Fetch mentioned above uses `WICG` but [WebTransport](https://wicg.github.io/web-transport/) uses `wicg`.

Reply:
I care more about the real name of the org than the way specs refer to it; we could either hardcode the names (there are only so many of them), or ask the info from the github API?

Reply:
Asking the info from the GitHub API would be nice. Is there a way to ask for all info at once? Otherwise, we'll end up with >300 requests to the API, which would exceed the [allowed limit](https://developer.github.com/v3/#rate-limiting) and require authentication.

Reply:
I think we only need to ask about organization names, not repositories, so this should be about ~6 requests

Reply:
I rewrote the code to implement that logic. I note I didn't include tests on this specific part because, even with test that say send 3 requests to the GitHub API, it only takes 12 runs to be rate limited (and GitHub does enforce this limit).